airbyte-integrations/connectors/source-pardot/source_pardot/manifest.yaml

```diff
@@ -0,0 +1,739 @@
+version: 4.3.2
+
+type: DeclarativeSource
+
+check:
+  type: CheckStream
+  stream_names:
+    - Campaigns
+
+definitions:
+  streams:
+    Campaigns:
+      type: DeclarativeStream
+      name: Campaigns
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: campaign/version/4/do/query
+          http_method: GET
+          request_parameters:
+            format: json
+            created_after: "{{ format_datetime(config['start_date'], '%Y-%m-%dT%H:%M:%SZ') }}"
+            sort_by: id
+            sort_order: ascending
+          request_headers:
+            Pardot-Business-Unit-Id: '{{ config["pardot_business_unit_id"] }}'
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - result
+              - campaign
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestOption
+            inject_into: request_parameter
+            field_name: id_greater_than
+          pagination_strategy:
+            type: CursorPagination
+            cursor_value: '{{ response.get("campaign", {})[-1].get("id", {}) }}'
```

Comment:
One campaign per page?

Reply:
No. The idea is to get the `id` of the last campaign in the batch and inject it to the `id_greater_than` query parameter

Reply:
Ah, nice!